management_api_app/api/routes/workspace_templates.py

```diff
@@ -23,22 +23,18 @@ async def get_workspace_templates(workspace_template_repo: WorkspaceTemplateRepo
     return WorkspaceTemplateNamesInList(templateNames=workspace_template_names)
 
 
-@router.post("/workspace-templates", status_code=status.HTTP_201_CREATED,
-             response_model=WorkspaceTemplateInResponse,
-             name=strings.API_CREATE_WORKSPACE_TEMPLATES)
-async def create_workspace_template(workspace_template_create: WorkspaceTemplateInCreate, workspace_template_repo: WorkspaceTemplateRepository =
-                                    Depends(get_repository(WorkspaceTemplateRepository))) -> WorkspaceTemplateInResponse:
```

Comment:
we are not following the 80 col rule for linebreaks? :)

Reply:
IMHO that is a remnant of a time when screens were much smaller. Nowadays I think splitting the lines reduces readability.